KomaMRICore/src/simulation/Bloch/KernelFunctions.jl

```diff
@@ -0,0 +1,48 @@
+using KernelAbstractions: @index, @kernel
```

Comment:
Can you check if the newly added CPU backend adds codecov for the kernel?

Also, add the comments to exclude the coverage for the metal extension.

You may also delete the interpolation adapters from the GPU extensions.

Reply:
I don't think it does since on the CPU it forwards the call to cumtrapz in KomaMRIBase. I forgot about code coverage not working for Metal, so I added the comments to exclude.